team.html

```diff
@@ -339,19 +339,21 @@ <h2>Nakul Sharma</h2>
 
     <!--  team ends -->
 
-    <!-- footer starts -->
+  <!--Footer starts-->  
     <footer id="footer">
         <div class="container">
             <div class="row subscribe-box py-5">
                 <div class="col-lg-4 col-md-4 col-sm-12 col-12 offset-lg-1">
                     <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 </div>
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
+
+                    <form action="mailto:algophantom0101@gmail.com">
                     <form action="#">
```

Comment:
Remove this form action as you already added a new one.

Reply:
Removed this one.